goodconf/__init__.py

```diff
@@ -71,10 +71,17 @@ def _load_config(path: str) -> dict[str, Any]:
         yaml = ruamel.yaml.YAML(typ="safe", pure=True)
         loader = yaml.load
     elif ext == ".toml":
-        import tomlkit
+        try:
+            import tomllib
+
+            def load(stream):
+                return tomllib.loads(f.read())
+
+        except ImportError:
```

Comment:
Could you leave a comment here about the except being for versions of Python < 3.11 (so we can easily remove it when we no longer support those versions)?